scripts/token_checker.py

```diff
@@ -23,7 +23,8 @@ def __init__(self):
                             "fantom": "ftm-fantom",
                             "celo": "celo-celo",
                             "base": "base-base",
-                            "zksync": "zksync-zksync"
+                            "zksync": "zksync-zksync",
+                            "zora": "eth-ethereum"
```

Comment:
fyi @aalan3 @couralex6 @0xRobin 
we are forcing zora prices to look on ethereum blockchain in the prices CI test. users should be good to add coinpaprika ID values which exist on other blockchains, then just use the zora address value in the spell to join other spells downstream.

Reply:
not sure how to communicate with their team on if / when zora will be included in API